crux-metrics/src/crux/metrics.clj

```diff
@@ -18,28 +20,57 @@
                :deps #{:crux.node/node :crux.node/indexer :crux.node/bus :crux.node/kv-store}}})
 
 (def jmx-reporter
-  {::jmx-reporter {:start-fn (fn [{::keys [registry]} {::keys [jmx-reporter-opts]}]
-                               (doto (jmx/reporter registry
-                                                   (merge {:domain "crux.metrics"}
-                                                          jmx-reporter-opts))
-                                 jmx/start))
+  {::jmx-reporter {:start-fn (fn [{::keys [registry]} args]
+                               (jmx/start-reporter registry args))
+                   :args {:crux.metrics.dropwizard.jmx/domain {:doc "Add custom domain"
+                                                               :required? false
+                                                               :crux.config/type :crux.config/string-vector}
```

Comment:
it seems more customary in Java land to pass a dot-delimited string for these, we'd probably confuse someone with knowledge of JMX by splitting it into a vector